tests/foreman/cli/test_myaccount.py

```diff
@@ -1,64 +1,86 @@
 # -*- encoding: utf-8 -*-
 """Test class for Users CLI"""
 
-from robottelo.decorators import stubbed
+from fauxfactory import gen_string
+from robottelo.cli.base import CLIReturnCodeError
+from robottelo.cli.factory import make_user
+from robottelo.cli.user import User
+from robottelo.datafactory import invalid_emails_list
+from robottelo.decorators import stubbed, tier1
 from robottelo.test import CLITestCase
 
 
 class MyAccountTestCase(CLITestCase):
-    """Implements Users tests in CLI
+    """Implements My Account functionality tests in CLI"""
 
-    [1] Positive Name variations - Alpha, Numeric, Alphanumeric, Symbols,
-    Latin1, Multibyte, Max length,  Min length, Max_db_size, html, css,
-    javascript, url, shell commands, sql, spaces in name
-
-    [2] Negative Name Variations -  Blank, Greater than Max Length,
-    Lesser than Min Length, Greater than Max DB size
-    """
-
-    @stubbed()
+    @classmethod
+    def setUpClass(cls):
+        """Create a new user for all myaccount tests to not impact the default
+        user
+        """
+        super(MyAccountTestCase, cls).setUpClass()
+
+        login = gen_string('alphanumeric', 30)
+        password = gen_string('alphanumeric', 30)
+        cls.user = make_user({
+            'login': login,
+            'password': password,
+            'admin': '1',
+        })
+        User.foreman_admin_username = login
+        User.foreman_admin_password = password
+
+    @classmethod
+    def tearDownClass(cls):
+        """Restore User object to previous state"""
+        del User.foreman_admin_username
+        del User.foreman_admin_password
+
+        super(MyAccountTestCase, cls).tearDownClass()
+
+    @tier1
     def test_positive_update_first_name(self):
-        """@Test: Update Firstname in My Account
+        """Update Firstname in My Account
 
         @Feature: My Account - Positive Update
 
-        @Steps:
-        1. Update current User with all variations of Firstname in [1]
-
         @Assert: Current User is updated
-
-        @Status: Manual
         """
+        new_firstname = gen_string('alphanumeric')
```

Comment:
Don't need to. First/last name boundary values already covered in scope of `test_user.py`